src/test/java/org/broadinstitute/hellbender/tools/funcotator/FuncotatorIntegrationTest.java

```diff
@@ -1910,5 +1917,170 @@ public void testCanHandleSymbollicAlleleFuncotations() {
+                    inputVcfName,
+                    outputFile,
+                    referencePath,
+                    dataSourcesPath,
+                    referenceVersion,
+                    outputFormatType,
+                    true);
+            runCommandLine(arguments);
+        }
+
+     }
+
+    @DataProvider
+    public Object[][] provideForCheckIfAlreadyAnnotatedTestWrong() {
+        return new Object[][] {
+                {
+                        FuncotatorTestConstants.NON_TRIVIAL_DATA_VALIDATION_TEST_HG19_DATA_SET_1_EXPECTED_OUTPUT,
+                        b37Reference,
+                        FuncotatorTestConstants.REFERENCE_VERSION_HG19,
+                        FuncotatorTestConstants.FUNCOTATOR_DATA_SOURCES_MAIN_FOLDER,
+                        FuncotatorTestConstants.NON_TRIVIAL_DATA_VALIDATION_TEST_HG19_DATA_SET_1_EXPECTED_OUTPUT,
+
+                }
+        };
+    }
+
+
+    @Test(dataProvider = "provideForCheckIfAlreadyAnnotatedTestWrong", expectedExceptions = UserException.BadInput.class)
+    public void testCheckIfAlreadyAnnotatedIntegratedWrong(final String inputVcfName,
+                                                            final String referencePath,
+                                                            final String referenceVersion,
+                                                            final String dataSourcesPath,
+                                                            final String expectedOutputPath ) {
+
+        for ( final FuncotatorArgumentDefinitions.OutputFormatType outputFormatType :
+                FuncotatorArgumentDefinitions.OutputFormatType.values()) {
+            if (outputFormatType.equals(FuncotatorArgumentDefinitions.OutputFormatType.SEG)) {
+                continue;
+            }
+            final String typeCorrectedExpectedOutPath = FilenameUtils.removeExtension(expectedOutputPath) +
+                    "." + outputFormatType.toString().toLowerCase();
+            final File outputFile = createTempFile(tmpOutDir + File.separator + inputVcfName + ".funcotator", "." + outputFormatType.toString().toLowerCase());
+
+            final ArgumentsBuilder arguments = createBaselineArgumentsForFuncotator(
+                    inputVcfName,
+                    outputFile,
+                    referencePath,
+                    dataSourcesPath,
+                    referenceVersion,
+                    outputFormatType,
+                    true);
+            runCommandLine(arguments);
+        }
+
+    }
+
+    /** In this test, we are giving as input a vcf which has already been annotated.
+     * Because we are not giving the flag argument which indicates that we should reannotate,
+     * then the default argument will be used which is to no reannotate and instead to throw
+     * an error.
+     */
+    @Test(dataProvider = "provideForCheckIfAlreadyAnnotatedTestWrong", expectedExceptions = UserException.BadInput.class)
```

Comment:
This test case and the previous one appear to be identical -- in both of them, you are running on an already-annotated file, with the `--reannotate-vcf` option set to `false`, and expecting an exception